wallet/src/wallet/tests.rs

```diff
@@ -5924,4 +5924,363 @@ fn create_order_fill_partially_conclude(#[case] seed: Seed) {
         );
     }
 }
-// create order, fill partially, conclude
+
+#[rstest]
+#[trace]
+#[case(Seed::from_entropy())]
+fn conflicting_delegation_account_nonce(#[case] seed: Seed) {
```

Comment:
Not sure this test is correct, if I understand correctly,
the spend_from_delegation_tx_2 depends on spend_from_delegation_tx_1,
so when spend_from_delegation_tx_1 is confirmed spend_from_delegation_tx_2 should still be valid not conflicting.

To be conflicting you need to create 3 txs, X1, and X2 that have nonce 0, and than add X2 as unconfirmed, and create Y that depend on X2 with nonce 1. Then confirm X1 which will conflict with X2 and Y.